packages/components/src/components/relative-datepicker/relative-datepicker.tsx

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { toMoment, daysFromTodayTo } from '@deriv/shared';
 import Text from '../text';
 
-const RelativeDatepicker = ({ onChange, min_date, max_date, title }) => {
-    const hidden_input_ref = React.useRef();
+type TRelativeDatepickerProps = {
+    max_date?: number | string;
+    min_date?: number | string;
+    onChange: (days_from_today_to: number) => void;
+    title: string;
+};
+
+const RelativeDatepicker = ({ onChange, min_date = 0, max_date, title }: TRelativeDatepickerProps) => {
+    const hidden_input_ref = React.useRef<HTMLInputElement>(null);
 
     const clickHandler = () => {
-        hidden_input_ref.current.click();
+        hidden_input_ref?.current?.click();
     };
-    const onChangeHandler = e => {
-        onChange(daysFromTodayTo(e.target.value));
+    const onChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+        onChange(+(daysFromTodayTo(e.target.value) ?? 0));
     };
 
-    const min_date_moment = toMoment()
-        .add(min_date || 0, 'd')
-        .format('YYYY-MM-DD');
-    const max_date_moment = max_date ? toMoment().add(max_date, 'd').format('YYYY-MM-DD') : null;
+    const min_date_moment = toMoment().add(min_date, 'd').format('YYYY-MM-DD');
```

Comment:
as I understand, when we pass here 'd', means 'days', then the first parameter has to be number. we can specify it in types only with 'number'
check please here:
https://momentjscom.readthedocs.io/en/latest/moment/03-manipulating/01-add/

Reply:
@yauheni-kryzhyk-deriv, the function signature also allows `strings`, so I didn't want to restrict that. Seeing that the previous implementation also allowed for the use of strings

Reply:
I understand your point. As I see in the code, we are passing there via props only numbers. so I believe there is no concerns to stay with one exact type :) 
how do you think?

Reply:
Alright. I'll update it